mllib-dal/build.sh

```diff
@@ -11,9 +11,14 @@ if [[ -z $(which mvn) ]]; then
  exit 1
 fi
 
-if [[ -z $DAALROOT ]]; then
- echo DAALROOT not defined!
- exit 1
+if [[ -e $DAALROOT ]]; then
+    export ONEDAL_VERSION=$(echo "$DAALROOT" | awk -F '/' '{print $(NF)}')
+elif [[ -e $DALROOT  ]]; then
+    export ONEDAL_VERSION=$(echo "$DALROOT" | awk -F '/' '{print $(NF)}')
+    DAALROOT=$DALROOT
```

Comment:
I thought we use DAALROOT for Makefile_2023.2.0 and DALROOT for Makefile right? why should assign DALROOT to DAALROOT?